core-backend/sandbox/src/funcs.rs

```diff
@@ -320,9 +303,9 @@ where
             let read_inheritor_id = ctx.register_read_decoded(inheritor_id_ptr);
             let inheritor_id = ctx.read_decoded(read_inheritor_id)?;
 
-            ctx.ext.exit().map_err(ActorSyscallFuncError::Core)?;
+            ctx.ext.exit()?;
```

Comment:
move above to be able drop execution due to insufficient gas before reading data from memory

Reply:
Done